fenix/app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -107,8 +119,10 @@ fun PrimaryButton(
         backgroundColor = backgroundColor,
         modifier = modifier,
         icon = icon,
+        iconModifier = iconModifier,
         tint = FirefoxTheme.colors.iconActionPrimary,
         onClick = onClick,
+        enabled = buttonEnabled,
```

Comment:
We want to reorder below `modifier` after making the changes above.

Reply:
Done